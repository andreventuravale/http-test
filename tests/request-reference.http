@@host=https://jsonplaceholder.typicode.com
@@dash={{$guid}}

# @name sample1
# @ignoreHeaders .*
GET {{host}}/todos/1?_={{dash}}
content-type: application/json

###

# @title Replay the sample1 request
# @ignoreHeaders .*
GET {{host}}/todos/{{sample1.$.response.body.id}}?_={{dash}}
content-type: application/json
