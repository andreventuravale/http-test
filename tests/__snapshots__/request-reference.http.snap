// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/home/avale/node/jest-dot-http-files/tests/request-reference.http GET https://jsonplaceholder.typicode.com/todos/1?_=f6677c2a-57ee-4a4c-8601-90d9a6655545 1`] = `
{
  "request": {
    "body": "undefined",
    "headers": [
      [
        "content-type",
        "application/json",
      ],
    ],
    "meta": {
      "ignoreHeaders": {
        "global": false,
        "value": ".*",
      },
      "name": {
        "global": false,
        "value": "sample1",
      },
    },
    "method": "GET",
    "url": "https://jsonplaceholder.typicode.com/todos/1?_=f6677c2a-57ee-4a4c-8601-90d9a6655545",
    "variables": {
      "dash": {
        "global": true,
        "value": "f6677c2a-57ee-4a4c-8601-90d9a6655545",
      },
      "host": {
        "global": true,
        "value": "https://jsonplaceholder.typicode.com",
      },
    },
  },
  "response": {
    "body": {
      "completed": false,
      "id": 1,
      "title": "delectus aut autem",
      "userId": 1,
    },
    "headers": [
      [
        "access-control-allow-credentials",
        Anything,
      ],
      [
        "alt-svc",
        Anything,
      ],
      [
        "cache-control",
        Anything,
      ],
      [
        "cf-cache-status",
        Anything,
      ],
      [
        "cf-ray",
        Anything,
      ],
      [
        "connection",
        Anything,
      ],
      [
        "content-encoding",
        Anything,
      ],
      [
        "content-type",
        Anything,
      ],
      [
        "date",
        Anything,
      ],
      [
        "etag",
        Anything,
      ],
      [
        "expires",
        Anything,
      ],
      [
        "nel",
        Anything,
      ],
      [
        "pragma",
        Anything,
      ],
      [
        "report-to",
        Anything,
      ],
      [
        "reporting-endpoints",
        Anything,
      ],
      [
        "server",
        Anything,
      ],
      [
        "transfer-encoding",
        Anything,
      ],
      [
        "vary",
        Anything,
      ],
      [
        "via",
        Anything,
      ],
      [
        "x-content-type-options",
        Anything,
      ],
      [
        "x-powered-by",
        Anything,
      ],
      [
        "x-ratelimit-limit",
        Anything,
      ],
      [
        "x-ratelimit-remaining",
        Anything,
      ],
      [
        "x-ratelimit-reset",
        Anything,
      ],
    ],
    "status": 200,
    "statusText": "OK",
  },
}
`;

exports[`/home/avale/node/jest-dot-http-files/tests/request-reference.http Replay the sample1 request 1`] = `
{
  "request": {
    "body": "undefined",
    "headers": [
      [
        "content-type",
        "application/json",
      ],
    ],
    "meta": {
      "ignoreHeaders": {
        "global": false,
        "value": ".*",
      },
      "title": {
        "global": false,
        "value": "Replay the sample1 request",
      },
    },
    "method": "GET",
    "url": "https://jsonplaceholder.typicode.com/todos/1?_=730c619a-9eab-4e66-89bd-1ec0f0d33dbc",
  },
  "response": {
    "body": {
      "completed": false,
      "id": 1,
      "title": "delectus aut autem",
      "userId": 1,
    },
    "headers": [
      [
        "access-control-allow-credentials",
        Anything,
      ],
      [
        "alt-svc",
        Anything,
      ],
      [
        "cache-control",
        Anything,
      ],
      [
        "cf-cache-status",
        Anything,
      ],
      [
        "cf-ray",
        Anything,
      ],
      [
        "connection",
        Anything,
      ],
      [
        "content-encoding",
        Anything,
      ],
      [
        "content-type",
        Anything,
      ],
      [
        "date",
        Anything,
      ],
      [
        "etag",
        Anything,
      ],
      [
        "expires",
        Anything,
      ],
      [
        "nel",
        Anything,
      ],
      [
        "pragma",
        Anything,
      ],
      [
        "report-to",
        Anything,
      ],
      [
        "reporting-endpoints",
        Anything,
      ],
      [
        "server",
        Anything,
      ],
      [
        "transfer-encoding",
        Anything,
      ],
      [
        "vary",
        Anything,
      ],
      [
        "via",
        Anything,
      ],
      [
        "x-content-type-options",
        Anything,
      ],
      [
        "x-powered-by",
        Anything,
      ],
      [
        "x-ratelimit-limit",
        Anything,
      ],
      [
        "x-ratelimit-remaining",
        Anything,
      ],
      [
        "x-ratelimit-reset",
        Anything,
      ],
    ],
    "status": 200,
    "statusText": "OK",
  },
}
`;
